.github/workflows/require-tests.yml

```
@@ -0,0 +1,17 @@
+name: 'Require Tests'
+
+on:
+  pull_request_target:
+    types: [opened]
+
+jobs:
+  check:
+    runs-on: ubuntu-latest
```

To be consistent with the other workflows, please use a specific Ubuntu version. This avoids sudden breakage when a new version becomes `ubuntu-latest`.

```suggestion
    runs-on: ubuntu-22.04
```